client/src/components/LandingItems.vue

```diff
@@ -0,0 +1,61 @@
+<template>
```

Comment:
This should be on another branch.  It's a diff card on the board.  You should only really work on one at a time.  That way we can work around each other.  One card = one branch.